src/import/instruments.ts

```diff
@@ -86,6 +86,15 @@ function getWeight(deepCopyRow: any): number {
         return 1000 * value
       case 'min':
         return 1000 * value
```

Comment:
Unrelated to this PR: This line looks wrong to me. Should this be `60 * 1000 * value` instead?

Reply:
Hi @evanw!

Yep this, is definitely wrong 😬. If you wouldn't mind updating this one and the snapshot while you're here, I'll hit merge afterwards

Reply:
I updated it but it appears to not be covered by any existing tests, so no snapshots needed to be changed.